website/docs/TypeScript.md

```diff
@@ -123,6 +123,27 @@ const config: Options.WebdriverIO = {
 }
 ```
 
+## Missing Types
+
+When using Node 20 or above, wdio runs ts-node with different settings as this is required in order to keep things running.
```

Comment:
I would suggest to not use `wdio` and write out the whole project name `WebdriverIO` for various reasons including search engine optimizations.